server/models/user.model.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+	email: {
+		type: String,
+		required: true,
+		unique: true,
+	},
+	password: {
+		type: String,
+		required: true,
+	},
+	role: {
+		type: String,
+		required: true,
+	},
+});
+
+const User = mongoose.model("user", userSchema);
```

Comment:
For Mongo the convention is generally to capitalize model names, like "User"

Reply:
Got it. Thank you.